loki.config.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   diffingEngine: "looks-same",
-  storiesFilter: "static-viz",
+  storiesFilter: "static-viz|viz",
```

Comment:
I think the stories filter is a regex, so I'm not sure if we need a `^viz|static-viz` instead